crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -180,6 +180,12 @@ def transition(inst, name, state)
         xs <=> ys
       end
 
+      # Make sure pacemaker is the first to execute on node delete
+      if state == "delete"
+        index = roles.index { |role| role.name.include? "pacemaker" }
```

Comment:
There are actually more than one role with "pacemaker" in the name. I'm not sure what was the original intention here.
Maybe search for "pacemaker-cluster-member" if that's what you want.

Reply:
Actually you're right, it should be pacemaker-config-<inst>. Will make the changes